src/server/chromium/crPage.ts

```diff
@@ -1007,7 +1007,11 @@ class FrameSession {
     if (this._crPage._browserContext._browser.isClank())
       return;
     const colorScheme = this._page._state.colorScheme || this._crPage._browserContext._options.colorScheme || 'light';
-    const features = colorScheme ? [{ name: 'prefers-color-scheme', value: colorScheme }] : [];
+    const reducedMotion = this._page._state.reducedMotion || this._crPage._browserContext._options.reducedMotion || 'no-preference';
+    const features = [
+      { name: 'prefers-color-scheme', value: colorScheme },
+      { name: 'prefers-reduced-motion', value: reducedMotion },
```

Comment:
How does this reset to system default? Or do we always want 'no-preference' by default?

Reply:
Passing null should reset it to system defaults. Hard to test since browsers implement the detection differently, some do it only for windows and some not.
I will test it and keep you updated.